timemachine/md/minimizer.py

```diff
@@ -140,85 +142,181 @@ def minimize_host_4d(
     ff: ff.Forcefield
         Wrapper class around a list of handlers
 
-    mol_coords: list of np.ndarray
+    mol_coords: list of np.ndarray, optional
         Pre-specify a list of mol coords. Else use the mol.GetConformer(0)
 
-    windows: integer
-        Number of lambda windows to lower the mols into the host via 4D decoupling
+    minimizer_steps_per_window: integer
+        Number of steps to run each window of the FIRE minimizer with
 
-    n_steps_per_window: integer
-        Number of steps to evaluate at each window
+    minimizer_windows: integer
+        Number of windows to run FIRE minimizer over
+
+    equilibration_steps: integer
+        Number of steps to run MD with
+
+    pressure: float
+        Pressure to run barostat with
+
+    temperature: float
+        Temperature to run MD with
+
+    barostat_interval: integer
+        How often to run the barostat
+
+    seed: integer
+        Seed for barostat and integrator
```

Comment:
some of these can be edited to provide more information not already present in parameter name, e.g. 
```suggestion
    pressure: float
        in bar (used by barostat)

    temperature: float
        in kelvin (used by integrator and barostat)

    barostat_interval: integer
        # of MD steps between barostat moves

    seed: integer
        integrator uses `seed`, barostat uses `seed+1`
```